get_instagram.py

```diff
@@ -0,0 +1,31 @@
+from InstagramAPI import InstagramAPI
+from key import user_name, password
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+        self.api = InstagramAPI(username=user_name, password=password)
+        self.api.login()
+        self.user_id = self.api.username_id
+
+    def get_followers(self):
+        followers_list = list()
+        followers = self.api.getTotalFollowers(self.user_id)
+        for follower in followers:
```

Comment:
This can be simplified with list comprehension.

Reply:
done